server/views/court-case-progress.njk

```diff
@@ -0,0 +1,46 @@
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h2 class="govuk-heading-m" id="caseHearingsHeading">Case progress</h2>
+        {% if data.hearings | length %}
+            <form name="previousCommentsForm" id="caseHearingsForm" method="post" action="summary/hearings/{{ "hideOlderHearings" if session.showPreviousHearings === data.caseId else "showPreviousHearings"}}">
+                {% if session.showPreviousHearings === data.caseId %}
+                    {% set limitPreviousHearings = data.hearings | length %}
+                {% else %}
+                    {% set limitPreviousHearings = 6 %}
+                {% endif %}
+                {% for hearing in data.hearings | limit(limitPreviousHearings) %}
+                <div class="app-summary-card govuk-!-margin-bottom-5">
+                    <header class="app-summary-card__header">
+                        <div class="app-summary-card__title">
+                            <span class="govuk-heading-s govuk-!-margin-bottom-0">{{ hearing.hearingTypeLabel }}</span>
+                            <span class="govuk-body">
+                                {{ hearing.hearingDateTime | caseCommentTimeFormat }}, Court {{ hearing.courtRoom }}, {{ hearing.session | lower }} session, {{ hearing.court }}
+                            </span>
+                        </div>
+                    </header>
+
+                    <div class="app-summary-card__body">
+                        <details class="govuk-details govuk-!-margin-bottom-0" data-module="govuk-details">
+                            <summary class="govuk-details__summary">
+                                <span class="govuk-details__summary-text">Add a note</span>
+                            </summary>
+                            <div class="govuk-details__text">
+                                <p class="govuk-!-margin-bottom-0">Coming soon...</p>
```

Comment:
@sarahyjja this is where the comments will go I think - @rschallapalli FYI @sarahyjja will pick up the front end work for this so it might be worth you two discussing any prior work you've done, or ideas you have about how we should go about that.